### DISCUSSION TITLE
How to change the filename of a generated svg

### DISCUSSION THREAD

**Discussion by ghost** [lowlighter/metrics#456] [Q&A] [answered]
Hi. How can you change the filename of a generated svg? I have the following code, but it says 'filename: metrics.activity.svg' is invalid yml code.

https://paste.ubuntu.com/p/SbbJDVjTp9/

**Comment by Nixinova** [comment, marked as answer]
It is invalid: 
```
	  filename: 'metrics.activity.svg'
```
It is indented with `<tab><space><space>`. Replace the tab with spaces to fix.

**Reply by ghost** [reply]
Thanks, it worked.
